cli/core/products/app/sync.py

```
@@ -0,0 +1,209 @@
+        merged: DataCollectionModel | None = None
+        for scope_name, service in scopes:
+            status.update(f"Create {scope_name} parameters for product {product_id}...")
+            service.set_new_parameter_group(parameter_group_collection)
+            scope_collection = service.create().collection
+            if merged is None:
+                merged = scope_collection
+            elif scope_collection is not None:
+                merged.add(scope_collection.collection)
+        return merged
+
+    def _run_create(self, status: Status) -> None:  # noqa: WPS210,WPS213
```

Ruff (0.15.12)

[warning] Invalid rule code in `# noqa`: WPS210, WPS213

Add non-Ruff rule codes to the `lint.external` configuration option

(RUF102)